src/set_container_metadata_request_base.cpp

```diff
@@ -0,0 +1,42 @@
+#include "set_container_metadata_request_base.h"
+
+#include "constants.h"
+#include "utility.h"
+
+namespace azure {  namespace storage_lite {
+
+    void set_container_metadata_request_base::build_request(const storage_account &a, http_base &h) const
+    {
+        const auto &r = *this;
+
+        h.set_absolute_timeout(5L);
+
+        h.set_method(http_base::http_method::put);
+
+        storage_url url = a.get_url(storage_account::service::blob);
+        url.append_path(r.container());
+
+        url.add_query(constants::query_restype, constants::query_restype_container);
+        url.add_query(constants::query_comp, constants::query_comp_metadata);
+        add_optional_query(url, constants::query_timeout, r.timeout());
+        h.set_url(url.to_string());
+
+        storage_headers headers;
+
+        add_content_length(h, headers, 0);
+        add_ms_header(h, headers, constants::header_ms_client_request_id, r.ms_client_request_id(), true);
+        // les is not supported.
```

Comment:
>les [](start = 11, length = 3)

Minor: lease?

Reply:
ditto

Reply:
fixed in https://github.com/Azure/azure-storage-cpplite/pull/50/commits/d847518b0bbe79742f41e51c6536979a58fce905